plugins/woocommerce/includes/wc-deprecated-functions.php

```diff
@@ -95,8 +95,22 @@ function wc_caught_exception( $exception_object, $function = '', $args = array()
 	$message .= '. Args: ' . print_r( $args, true ) . '.';
 
 	do_action( 'woocommerce_caught_exception', $exception_object, $function, $args );
+
 	error_log( "Exception caught in {$function}. {$message}." );
 	// @codingStandardsIgnoreEnd
+
+	// Logging error in wc logs
+	$logger = wc_get_logger();
+	$logger->error(
+		sprintf(
+			$exception_object->getMessage() . ' Order #%d',
+			$args[0]
+		),
+		array(
+			'order' => $args[0],
+			'error' => $exception_object,
+		)
+	);
```

Comment:
@sumughdas what do you think of compressing this down a bit, and also removing the assumption it will be used only for order data?

```suggestion
	wc_get_logger()->error( $message, array( 'function' => $function ) );
```

Variable `$message` is already defined a little higher and contains the exception message as well as the information in the `$args` array: the key thing here is that this function is invoked in various other contexts, too [(example)](https://github.com/woocommerce/woocommerce/blob/7.7.0/plugins/woocommerce/src/Admin/Note/Notes.php#L67), and not just in relation to orders.